src/openrct2/actions/RideCreateAction.hpp

```diff
@@ -310,5 +277,41 @@ struct RideCreateAction : public GameActionBase<GAME_COMMAND_CREATE_RIDE, GA_FLA
 
         return std::move(res);
     }
-};
 
+private:
+    sint32 GetSubType() const
+    {
+        sint32 subType = rideSubType;
+
+        if (subType == RIDE_ENTRY_INDEX_NULL)
+        {
+            uint8 *availableRideEntries = get_ride_entry_indices_for_ride_type(rideType);
+            for (uint8 *rei = availableRideEntries; *rei != RIDE_ENTRY_INDEX_NULL; rei++)
```

Comment:
Use a better name than rei.

Reply:
i'd just use "i"

Reply:
Thats worse, its a pointer if you haven't noticed.

Reply:
I mean, rei looks like it stands for Ride Entry Iterator, or Ride Entry Index. But I've always stuck with short variable names for iteration sequences just to tell if part of a code is part of some form of iteration or loop. maybe rideEntryIndex?